### ISSUE TITLE
Clicks are slow, can probably fix with Fastclick

### ISSUE THREAD

**Issue by richiesun** [dgileadi/zepto-page-transitions#2]
Page transitions have a slow hang after clicking. This is default for webview clicks, but I found another plugin that eliminates that issue and speeds up the click resulting in an overall boost in UI performance. 

Here is the Git Rep: https://github.com/ftlabs/fastclick

Here is more info from the original author: http://maxogden.com/fast-webview-applications.html

Any chance of getting your plugin to play nice with fast click? I think it would be a worthy effort. Not really sure how to go about integrating the two.

**Comment by dgileadi** [comment]
I suspect you could catch the 'pageshow' event this plugin generates, something like:

$(document).on('pageshow', function(e, page) {
  new FastClick(page);
});

I haven't tested this, though.  I'll take a look at figuring out a more general method for fixing slow-clicks in any case, even if it just ends up being to recommend FastClick.

**Comment by richiesun** [comment]
Hey David,

Thanks for quick reply. Seems like Zepto plugins are just starting to be developed. You are def ahead of the pack with the page transitions. Will definitely be watching this Rep. 

I've already tested the FastClick plugin and it works as advertised. I've tested both your plugin and fastclick in the same project, but the two aren't talking to each other. Really interested in what you come up with. 

Thanks,
Richie
